content/docs/faq-functions.md

```diff
@@ -200,31 +200,37 @@ class Alphabet extends React.Component {
       justClicked: e.target.dataset.letter
     });
   }
-  
+
   render() {
     return (
       <div>
         Just clicked: {this.state.justClicked}
         <ul>
-          {this.state.letters.map(letter => 
+          {this.state.letters.map(letter =>
             <li key={letter} data-letter={letter} onClick={this.handleClick}>
               {letter}
             </li>
           )}
         </ul>
       </div>
-   )
+    )
   }
 }
 ```
 
 ### How can I prevent a function from being called too quickly or too many times in a row?
 
-If you have an event handler such as `onClick` or `onScroll` and want to prevent the callback from being fired too quickly, you can wrap the handler with a utility such as [`_.debounce`](https://lodash.com/docs#debounce) or [`_.throttle`](https://lodash.com/docs#throttle). See [this visualization](http://demo.nimius.net/debounce_throttle/) for a comparison of the two.
+If you have an event handler such as `onClick` or `onScroll` and want to prevent the callback from being fired too quickly, then you can limit the rate at which callback is executed. This can be done by using:
+
+- **throttling**: sample changes based on a time based frequency ([`_.throttle`](https://lodash.com/docs#throttle))
+- **debouncing**: publish changes after a period of inactivity ([`_.debounce`](https://lodash.com/docs#debounce))
+- **`requestAnimationFrame` throttling**: sample changes based on [`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) ([`raf-schd`](https://github.com/alexreardon/raf-schd))
```

Comment:
I like this change ^ 😄 

Maybe we could improve clarity though that the NPM modules we link to are only _one_ option (not the only one). Maybe something like:

- **Debouncing**: Publish changes after a period of inactivity (eg [`_.debounce`](https://lodash.com/docs#debounce))
- **Timer-based throttling**: Sample changes based on a time based frequency (eg [`_.throttle`](https://lodash.com/docs#throttle))
- **`requestAnimationFrame` throttling**: Sample changes based on [`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) (eg [`raf`](https://www.npmjs.com/package/raf), [`raf-schd`](https://www.npmjs.com/package/raf-schd))

Reply:
As mentioned below, `raf` is just a polyfill and is not a higher level utility for sampling with `requestAnimationFrame`